ceph.spec.in

```diff
@@ -2138,6 +2138,12 @@ fi
 %dir %{_localstatedir}/lib/ceph/radosgw
 %{_unitdir}/ceph-radosgw@.service
 %{_unitdir}/ceph-radosgw.target
+%exclude %{_includedir}/cpp_redis
```

Comment:
@samarahu could you expiriment with removing the changes in this file and seeing if the rpms can still be built? 

I think maybe the `%exlude /usr/lib/lib{cpp_redis,tacopie}.a` might still be necessary. So if removing everything fails maybe try adding those 2 lines back in.